opensrp-chw-core/src/main/java/org/smartregister/chw/core/activity/CoreChildProfileActivity.java

```diff
@@ -402,6 +404,14 @@ public void setVisitButtonDueStatus() {
         textViewRecord.setTextColor(getResources().getColor(R.color.white));
     }
 
+    @Override
+    public void togglePhysicallyDisabled(boolean show) {
+        if (show) {
```

Comment:
App versions that do not have the physically challenged text view , i.e those that override the profile xml will crash at this line